src/main/kotlin/test/project/findValue/database/repository/FileRepository.kt

```diff
@@ -0,0 +1,21 @@
+package test.project.findValue.database.repository
+
+import io.quarkus.hibernate.orm.panache.kotlin.PanacheRepository
+import test.project.findValue.database.dto.FileEntity
+import javax.enterprise.context.ApplicationScoped
+import javax.transaction.Transactional
+
+@Transactional
+@ApplicationScoped
+class FileRepository : PanacheRepository<FileEntity> {
+    fun findByName(name: String) = find("name", name).firstResult()
+    fun deleteFile() = delete("name", "Stef")
+    fun findByValue(valueId: String): MutableSet<String> {
+        val fileNames: MutableSet<String> = mutableSetOf()
+        find("from File as f, Value as v, ValueInFiles as fiv" +
```

Comment:
Why not just to return 
`fun findByValue(valueId: String): Set<String>{
          find(....);
}`
?